Emip miracles of jesus work files 22 march 2021

Ralph-Lee-UK opened the pull request:
Here is the rather large PR for the Miracles of Jesus. I have done the following:

Added bibliographic details and incipits and explicits from Grebaut to existing MJ work files. 
Created new MJ work files for those that didn't exist, adding incipits and explicits from Grebaut where they are available, and including the incipit as recorded in the EMIP catalogue where not available. 
Some miracles were only in van Rompey and Arras, for which I do not have the text, so in these cases only the biographical information could be recorded, but I included the incipit from EMIP02392
Each new entry is recorded as forming part of LIT2382Taamme

Added textparts and corresp references to LIT2382Taamme

There is a lot of work here. Perhaps you could first check for systematic errors in the formatting of the files, and ways in which they don't match the BM guidelines. I can then correct these, before perhaps a final check. I am not always as accurate as I should be in transcribing text, but most of the transcription was done by Jeremey Brown who is extremely accurate in transcribing text! 

Don't hesitate to push work back on me to get this as good as it can be.

Ralph-Lee-UK commented:
@thea-m - sorry that I put so much work on you. Is there any way that I can help to expedite this approval?

thea-m commented:
No, I apologise for the delay! Everything is fine in general, and thank you for this throurough and extremely valuable work! I just didn't finish reading all the Geez yet. I'll publish now the comments that I have (until file LIT6397MJVirginityMary), so that if you want to work on it, you can continue in parallel with me reading the rest (mainly, one bibliography issue). Thank you again!

Ralph-Lee-UK commented:
@thea-m this is great. I will get on with these tomorrow, and look out for any further comments.

Ralph-Lee-UK commented on 1001-2000/LIT1111Anqasa.xml:
```diff
@@ -1,12 +1,15 @@
             <title xml:lang="gez" type="normalized" corresp="#t1">ʾAnqaṣa nǝssǝḥa</title>
             <title xml:lang="en" corresp="#t1">The Gate of Penance</title>
+            <title xml:lang="gez" xml:id="t2">ጸሎተ፡ ንስሐ፡ ዘተቀድሐ፡ እም፹ወ፩ መጸሕፍት፡</title>
+            <title xml:lang="gez" type="normalized" corresp="#t2">ṣälotä nǝsǝḥa zä-täqädǝḥa ʾǝm̄-sämānǝyā wä-ʾaḥadu mäṣäḥǝft</title>
```
Done, my mistake!

Ralph-Lee-UK commented on 4001-5000/LIT4720Miracle.xml:
```diff
@@ -50,21 +51,47 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+              <div type="textpart" subtype="incipit" xml:lang="gez">
+                <bibl><ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">827-828</citedRange></bibl>
+                <ab>
+                  ወእንዘ፡ የኃልፍ፡ እግዚእ፡ ኢየሱስ፡ በሀገረ፡ ናይን፡ ወምስለ፡ አርዳኢሁ፡ ስም
```
@PietroLiuzzo and @thea-m I have only put the line breaks in to make the transcription process easier to check. My understanding is that a line break on its own is not counted as a space. Is that correct? In the end once I have checked all the transcriptions once more then I could just remove the line breaks to avoid any confusion.

Ralph-Lee-UK replied:
I have now removed the line breaks

Ralph-Lee-UK commented on new/LIT6397MJVirginityMary.xml:
```diff
@@ -0,0 +1,99 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Miracle of Jesus: Concerning the Virginity of Mary</title>
```
Noted

Ralph-Lee-UK commented on new/LIT6017MJLion.xml:
```diff
@@ -78,7 +83,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                <bibl><ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">779-783</citedRange></bibl>
+                <ab>
+                  ወእምዝ፡ አኀዞ፡ ለዮሴፍ፡ እዴሁ፡ ወአዕረጎ፡ መልዕልተ፡ ደብረ፡ ታቦር፡ ወአ
+                  ርአዮ፡ ኃይሎ፡ በህየ፡ ዘኢርእየ፡ ቀድሚሙ፨ ወእምህየ፡ ተመይጡ፡ ሀገረ፡ ናዝሬት፨
```
Corrected

Ralph-Lee-UK commented on new/LIT6396MJIntroduction.xml:
```diff
@@ -0,0 +1,95 @@
+          <bibl><ptr target="bm:Grebaut1917MiraclesJesusI"/><citedRange unit="page">558</citedRange></bibl>
+          <ab>
+            ዝንቱ፡ ምሥጢር፡ አምላካዊ፡ ዘነገሮ፡ እግዚአብሔር፡ ወአምላክነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ለረድኡ፡ ዮሐንስ፡ ወልደ፡ ዘብዴዎስ፡ እምቅድመ፡ ዕርገቱ፡ ለእግዚእነ፡ ወአምላክነ፡
+            ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ውስተ፡ ሰማያት፡ ዘኢተፈልጠ፡ እምአቡሁ፡ ወእመንፈስ፡ ቅዱስ፡ አሐተኒ፡ ሰዓተ፡ ወኢክመ፡ ቅጽበተ፡ ዓይን።
```
Corrected

Ralph-Lee-UK commented on new/LIT6396MJIntroduction.xml:
```diff
@@ -0,0 +1,95 @@
+            ወሶበ፡ ተፈጸማ፡ እማንቱ፡ ዓመታት፡ እምፀዐቱ፡ ለአቡ
+            ነ፡ አዳም፡ እምገነት፡ ቀርበ፡ ጊዜ፡ ምጽአቱ፡ ለመድኀኒተ፡ ኵሉ፡ ዓለም፡ ወመድኀ
+            ኒቶሙ፡ እምቅኔሁ፡ ለሰይጣን፡ እስመ፡ ሞኦሙ፡ እስመ፡ አምለኩ፡ ጣዖተ፡ ኀዲጎ
+            ሙ፡ እግዚአብሔርሃ መስተሣህለ፨
```
Corrected

Ralph-Lee-UK commented:
@thea-m I believe that I have addressed the issues that you mention - the bibliographical refs where there are more than one, the 'en' title rather than 'gez', the line breaks that I had left in just to make copying easier, but which were unnecessary, and the various spelling errors that you highlighted. I'm sure there is more, but this is moving on nicely now!

thea-m commented on new/LIT6403MJResSarah.xml:
```diff
@@ -0,0 +1,86 @@
+              <div type="textpart" subtype="explicit" xml:lang="gez">
+                <bibl><ptr target="bm:Grebaut1923MiraclesJesusIII"/><citedRange unit="page">796-797</citedRange></bibl>
+                <ab>
+                  እምይእዜሰ፡ የአምኑ፡ ብየ፡ አሕዛብ፡ ነኪራነ፡ ነገድ፡ ህየንቲክሙ፡ ከመ፡ ይትፈጸም፡ ትንቢተ፡ዳዊት፡ ነቢይ፡ ዘይቤ፡ ሕዝብ፡ ዘይትፈጠር፡ ይሴብሖ፡ ለእግዚአብሔር። ወእምዝ፡ ይቤሎን፡ እግዚእ፡ ኢየሱስ፡ ለሳራ፡ ወለርብቃ፡ ወለራሄል፡ ተመየጣ፡ ውስተ፡ መካናቲክን፡ በሰላም። ወሶኔሃ፡ ተመይጣ፡ ውስተ፡ መቃብሪሆን፨
```
ወሶቤሃ?

Ralph-Lee-UK replied:
Corrected

thea-m commented on new/LIT6411MJMelons.xml:
```diff
@@ -0,0 +1,87 @@
+            <div type="textpart" subtype="incipit" xml:lang="gez">
+              <bibl><ptr target="bm:Grebaut1923MiraclesJesusIII"/><citedRange unit="page">798</citedRange></bibl>
+              <ab>
+                አሐተ፡ ዕለተ፡ ሖረ፡ እግዚእ፡ ኢየሱስ፡ በምድረ፡ ቂሳርያ፡ ዘፍልስጥኤም፡ ወምስ፡ አርዳኢሁ፡ ርእየ፡ ብእሴ፡ ዘይቀውም፡ ኀበ፡ ዘርአ፡ መልጰጶን፡ ዘይበኪ፡ ወይጐድእ፡ በእዴሁ፡ ገጾ፡ ወእንግድዓሁ። ወይቤሎ፡ እግዚእ፡ ኢየሱስ፡ ምንተ፡ ኮንከ፡ ወምንትኑ፡ ረከበከ፡ ኦብእሲ፡ ወለምንት፡ ትበኪ።
```
ወምስለ?

thea-m commented on new/LIT6418MJSermonMtOlives.xml:
```diff
@@ -0,0 +1,88 @@
+            <div type="textpart" subtype="incipit" xml:lang="gez">
+              <bibl><ptr target="bm:Grebaut1923MiraclesJesusIII"/><citedRange unit="page">834</citedRange></bibl>
+              <ab>
+                ወሖረ፡ እግዚእ፡ ኢየሱስ፡ ክርስቶስ፡ በሳኒታሁ፡ ድብረ፡ ዘይት፡ ወምስሌሁ፡ ዮሐንስ፡ ወያዕቆብ፡ ደቂቀ፡ ዘብዴዎስ፡ አርዳኢሁ። ወሶበ፡ ርእዩ፡ ያዕቆብ፡ ወዮሐንስ፡ ውስተ፡ ቤተ፡ መቅደስ፡ ይቤልዎ፡ ምንተ፡ ይሤኒ፡ ዝንቱ፡ ቢተ፡ መቅደስ። 
```
ቢተ፡ መቅደስ?

Ralph-Lee-UK replied:
Corrected

thea-m commented on new/LIT6420MJMarriageCana.xml:
```diff
@@ -0,0 +1,77 @@
+                  without subscription and supplications.</note>
+                  <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <ab>
+                      በእንተ፡ አመ፡ ረሰዮ፡ ለማይ፡ ወይነ።<gap reason="omitted"/> ወሀሎ፡ ፩ብእሲ፡ በሀገረ፡ ባቲላስ፡ እንተ፡ ይእቲ፡ ምድረ፡ ቃና፡ ዘገሊላ፡ ከብካበ፡ ለዘመደ፡ አትናኤል።
```
not ናትናኤል?

Ralph-Lee-UK replied:
The ms has አትናኤል. Should we mark this with with 'sic'
![EMIP02392_Weiner357_026](https://user-images.githubusercontent.com/39619238/121157509-884b5580-c841-11eb-9cbb-a7df88ffbe79.jpg)

thea-m replied:
No, I think it can stay as it is, I was just wondering whether it was a typo

thea-m reviewed (changes requested):
I'm so sorry for the terrible delay, I hope it won't happen again. But now it is done (just some minor spelling checks), and this really will be an amazing addition to BM!

Ralph-Lee-UK commented on new/LIT6402MJInfancy.xml:
```diff
@@ -0,0 +1,91 @@
+            <div type="edition">
+              <note>The following division into textparts and excerpts are taken from <ref type="mss" corresp="EMIP02392"/>,
+                  without subscription and supplications.</note>
+                  <label>The second recension of the Infancy Gospel</label>
```
According to the catalogue this is 'The second recension of the Infancy Gospel. Van Rompay and Arras 1975, no. 44.' which is broken down into three parts, Jesus rides a ray of sunshine, Jesus makes birds out of mud and orders them to fly away, Jesus mixes and separates the colors of his master's ink

thea-m replied:
Having only read the EAe article on the topic, it is not completely clear to me what is meant by this...In case you know it better and can quickly clarify it in a few words, that would be helpful, otherwise I think we can go ahead and trust the catalogue.

Ralph-Lee-UK commented on new/LIT6402MJInfancy.xml:
```diff
@@ -0,0 +1,91 @@
+                  <div type="textpart" xml:id="JesusInk">
+                    <label>Jesus mixes and separates the colors of his master's ink</label>
+                    <ab>
+                      ተአምሪሁ፡ ለእግዚእነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ዘገብረ፡ ለሊሁ፡ አመ፡ ለሊሁ፡ አመ፡ ልሕቀ፡ እግዚእነ፡ ኢየሱስ፡ ወሀብዎ፡ ለመምሕር፡ ከመ፡ ይትመሐር፡ መጽሐፈ።
```
The scribe does indeed mark this twice, so I have marked the second occurrence as 'surplus'

Ralph-Lee-UK commented:
@thea-m please don't apologise - this was a huge amount of work for you to check. I have made the simple corrections, and there remain two questions marked above - how to describe the infancy gospel, and the spelling of አትናኤል and how to mark that.
